src/sass/main.scss

```diff
@@ -1,5 +1,120 @@
 // scss goes here
 
 html {
-  font-family: helvetica;
+  font-family: sans-serif;
+}
+
+// used for theme skins with a default to gray
+@mixin color_cascade($color: #999) {
+  .wand-trigger {
+    border: 2px solid $color;
+    color: $color;
+  }
+  .wand-history-node {
+    border-color: $color;
+    &:before {
+      background-color: white;
+      border: 0.2em solid $color;
+    }
+  }
+  .wand-node-container {
+    border-color: $color;
+    &:before {
+      background-color: $color;
+      border: 0.2em solid $color;
+    }
+  }
+  .wand-history-title {
+    color: $color;
+  }
+}
+
+.wand {
```

Comment:
I’m down to chose something different, but I would stay away from weird casings like WaND — pain in the ass to type. I like underscores as a possibility.

On Sat, Aug 1, 2015 at 3:27 PM, Sam Matthews <notifications@github.com>
wrote:

>> +      border: 0.2em solid $color;
>> +    }
>> +  }
>> +  .wand-node-container {
>> +    border-color: $color;
>> +    &:before {
>> +      background-color: $color;
>> +      border: 0.2em solid $color;
>> +    }
>> +  }
>> +  .wand-history-title {
>> +    color: $color;
>> +  }
>> +}
>> +
>> +.wand {
> I'm totally fine with choosing something different - underscores are pretty safe if we want to do that. @phiden what do you think?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/codeforamerica/wand/pull/32/files#r36037645